Frontend/ui-customer/src/Components/MessagingView/View.purs

```diff
@@ -0,0 +1,518 @@
+   , linearLayout
+      [ height $ V 48
+      , width MATCH_PARENT
+      , padding (PaddingHorizontal 16 8)
+      , cornerRadius 24.0
+      , accessibility DISABLE
+      , gravity CENTER_VERTICAL
+      , background Color.blue600
+      , orientation HORIZONTAL
+      , afterRender push $ const $ NoAction
+      ][ editText $
+         [ weight 1.0
+         , height $ V 48
+         , id (getNewIDWithTag "ChatInputEditText")
+         , background Color.blue600
+         , cornerRadius 24.0
+         , accessibility ENABLE
+         , accessibilityHint $ "Message Driver : Text Input : Select to send message to driver" 
+         , hint $ config.hint <> " " <> fromMaybe "" ((STR.split (STR.Pattern " ") config.userConfig.userName) !! 0) <> "..."
+         , singleLine true
+         , hintColor Color.black700
+         , ellipsize true
+         , onChange push $ TextChanged
+         , pattern "[^\n]*,255"
+         ] <> FontStyle.body1 LanguageStyle
+       , linearLayout
+         [ height $ V 36
+         , width $ V 36
+         , gravity CENTER
+         , onClick push $ const $ SendMessage
+         , accessibilityHint "Send Message : Button"
+         , accessibility ENABLE
+         ][ imageView
+            [ imageWithFallback $ if config.feature.sendMessageActive then fetchImage FF_COMMON_ASSET "ic_send_blue" else fetchImage FF_COMMON_ASSET "ic_send"
+            , accessibility DISABLE
+            , height $ V 20 
+            , width $ V 20 
+            ] 
+         ]
+      ]
+  ]
+
+emptyChatView :: forall w. Config -> (Action -> Effect Unit) -> PrestoDOM (Effect Unit) w
+emptyChatView config push =
+  linearLayout
+  [ height $ V 150
+  , background Color.transparent
+  , width MATCH_PARENT
+  , accessibility DISABLE
+  ][]
+
+suggestionsView :: forall w. Config -> (Action -> Effect Unit) -> PrestoDOM (Effect Unit) w
+suggestionsView config push =
+  linearLayout
+  [ height WRAP_CONTENT
+  , width MATCH_PARENT
+  , afterRender push $ const $ NoAction
+  , visibility $ boolToVisibility $ not $ null config.chatSuggestionsList || not config.feature.canSendSuggestion || not config.feature.enableSuggestions
+  , background Color.white900
+  , accessibility DISABLE
+  , margin $ MarginBottom 16
+  ][ textView $
+     [ text $ getString QUICK <> "\n" <> getString REPLIES
+     , color Color.black700
+     , margin $ MarginRight 12
+     , accessibility ENABLE
+     , accessibilityHint $ "Quick Replies"
+     ] <> FontStyle.captions TypoGraphy
+   , horizontalScrollView
+     [ height WRAP_CONTENT
+     , weight 1.0
+     , scrollBarX false
+     , accessibility DISABLE
+     , disableKeyboardAvoidance true
+     ][ linearLayout
+        [ height WRAP_CONTENT
+        , width WRAP_CONTENT
+        , gravity LEFT
+        , accessibility DISABLE
+        ] (mapWithIndex (\index item -> quickMessageView config item index push)(config.chatSuggestionsList))
+      ]
+  ]
+
+dummyTextView :: forall w . PrestoDOM (Effect Unit) w
```

Comment:
Instead of textView try using linearlayout or make visibility false for this else it will take some space